pkg/controller/cloudfront/distribution/lateinit.go

```diff
@@ -48,6 +48,9 @@ func lateInitialize(dp *svcapitypes.DistributionParameters, gdo *svcsdk.GetDistr
 	in := dp.DistributionConfig
 	from := gdo.Distribution.DistributionConfig
 
+	// during import scenarios, we need the original caller reference info
```

Comment:
Could you add a comment here about what we discussed offline? i.e. the caller reference is not exposed to users who would like to import the resource, hence we need to derive it from the first response from AWS if user didn't give it explicitly.

Reply:
done